Makefile

```diff
@@ -419,13 +419,16 @@ endif
 
 ifeq ($(OS), WINNT)
 	[ ! -d dist-extras ] || ( cd dist-extras && \
-		cp 7z.exe 7z.dll libexpat-1.dll zlib1.dll libgfortran-3.dll libquadmath-0.dll libstdc++-6.dll libgcc_s_s*-1.dll libssp-0.dll $(bindir) && \
+	    cp 7z.exe 7z.dll libexpat-1.dll zlib1.dll libgfortran-3.dll libquadmath-0.dll libstdc++-6.dll libgcc_s_s*-1.dll libssp-0.dll $(bindir) && \
 	    mkdir $(DESTDIR)$(prefix)/Git && \
 	    7z x PortableGit.7z -o"$(DESTDIR)$(prefix)/Git" && \
 	    echo "[core] eol = lf" >> "$(DESTDIR)$(prefix)/Git/etc/gitconfig" && \
 	    sed -i "s/\bautocrlf = true$$/autocrlf = input/" "$(DESTDIR)$(prefix)/Git/etc/gitconfig" && \
 	    cp busybox.exe $(DESTDIR)$(prefix)/Git/bin/echo.exe && \
-	    cp busybox.exe $(DESTDIR)$(prefix)/Git/bin/printf.exe )
+	    cp busybox.exe $(DESTDIR)$(prefix)/Git/bin/printf.exe && \
+            mkdir -p $(DESTDIR)$(prefix)/mintty/usr/bin	&& \
```

Comment:
The `mintty` stuff gets put in its own directory because @ihnorton reported in https://github.com/JuliaLang/julia/issues/7267#issuecomment-84674766 that it was necessary that `mintty` exist in a `usr/bin` directory in order for `cygwin-console-helper` to do its thing (hiding the extra terminal window).